lib/rules/aws-ec2-instance-uses-security-group.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const debug = require('debug')('tfrules/aws-ec2-instance-uses-security-group');
```

Comment:
Would you like me to change the file name for the .js and .spec.js files as well?